src/content/docs/docs/guides/verifications.md

```diff
@@ -49,33 +55,43 @@ Vision verifications are significantly more expensive and slower, so resort to t
 Meta Llama 3.2 [does not support vision verifications][2] yet.
 :::
 
+:::note
+A screenshot is taken for the visible part of the page.
+:::
+
 Occasionally, the web page state is not enough for Alumnium to perform the check. In this case, instruct it to take a screenshot of the page and include it in the verification decision. This is useful when you need to check the visual representation of elements or their spatial relationships.
 
 For example, in your test for the To-Do application, you might need a check that a completed task is shown with a strikethrough style.
 
-![A screenshot of To Do application with one task completed](../../../../assets/todo-one-checked.png)
-
 ```python
-al.check("'buy bread' title font style is strikethrough")
+al.check("'buy milk' title font style is strikethrough")
 ```
 
 Without a screenshot, this assertion can fail because there is no indication of font style in the web page itself:
 
 ```bash wrap
-AssertionError: The ARIA tree does not indicate that the 'buy bread' item has a strikethrough style. It shows that the checkbox for 'buy bread' is checked, which typically indicates that the item is completed, but there is no mention of a strikethrough style in the provided ARIA tree.
+AssertionError: The ARIA tree does not provide any information regarding the font style of the 'buy milk' task, including whether it is strikethrough or not.
 ```
 
 To make the check more reliable, add a screenshot to it:
 
 ```python
-al.check("'buy bread' title font style is strikethrough", vision=True)
+al.check("'buy milk' title font style is strikethrough", vision=True)
 ```
 
-Keep in mind, that the screenshot is taken for the visible part of the page.
+<video class="rounded-xl" alt="A screen recording of Alumnium failing to determe font style of completed task" controls controlslist="nofullscreen" disablepictureinpicture muted playsinline width="100%" height="auto">
+  <source src="../../../src/assets/videos/check-vision.mp4" type="video/mp4" />
+  <source src="../../../src/assets/videos/check-vision.webm" type="video/webm" />
+</video>
 
 ## Flakiness
```

Comment:
For the future, this can probably be moved to a separate doc/article with more details